client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -151,7 +151,11 @@ UndocumentedSite.prototype.updateUser = function( userId, attributes, callback )
 	}, callback );
 };
 
-UndocumentedSite.prototype.getUser = function( login, callback ) {
+UndocumentedSite.prototype.getUser = function( userId, callback ) {
+	return this.wpcom.req.get( '/sites/' + this._id + '/users/' + userId, callback );
```

Comment:
maybe add string templates here?

```js
return this.wpcom.req.get( `/sites/${ this._id }/users/${ userId }`, callback );
```

Feel free to ignore this comment. It isn't a blocker.